libs/initScripts/local.js

```diff
@@ -159,15 +174,15 @@ const _registerCnode2 = async (audiusLibs, ethAccounts) => {
   await registerLocalService(audiusLibs2, spCreatorNodeType, creatorNodeEndpoint2, amountOfAuds)
 }
 
-// Account 4
+// Account 3
 const _registerCnode3 = async (audiusLibs, ethAccounts) => {
-  let audiusLibs2 = await initAudiusLibs(true, null, ethAccounts[4])
+  let audiusLibs2 = await initAudiusLibs(true, null, ethAccounts[3])
```

Comment:
i think this is [4] because discprov uses [3]. maybe we change the discovery index and reserve the first 10 for cnodes?

Reply:
Yeah that sounds right. Having the cnodes skip accounts from 2 to 4 was creating issues in mad-dog